<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>464</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>523</width>
    <height>464</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>523</width>
    <height>464</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hoop Chat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>44</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Parola</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>83</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Kullanıcı Adı</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Oturum Aç</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>İptal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>180</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Henüz bir hesabın yok mu?</string>
    </property>
   </widget>
   <widget class="QLabel" name="kayit_ol">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>55</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Kayıt ol!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Statics/hoop_logo.fw.png</pixmap>
    </property>
   </widget>
   <zorder>usernameBox</zorder>
   <zorder>passBox</zorder>
   <zorder>loginButton</zorder>
   <zorder>exitButton</zorder>
   <zorder>label_2</zorder>
   <zorder>kayit_ol</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHoop">
    <property name="title">
     <string>Hoop</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuYard_m">
    <property name="title">
     <string>Yardım</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionTeam"/>
   </widget>
   <addaction name="menuHoop"/>
   <addaction name="menuYard_m"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_exit">
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Hoop Hakkında</string>
   </property>
  </action>
  <action name="actionTeam">
   <property name="text">
    <string>Ekip</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
